### PR TITLE
Add max cpu alert description

### PR DESCRIPTION
**Pull request by qqqdan**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.3 (TiDB 7.3 versions)
- [x] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-cloud/monitor-built-in-alerting.md [hunk 1]
```diff
@@ -78,6 +78,7 @@ The following table provides the TiDB Cloud built-in alert conditions and the co
 >
 > Although these alert conditions do not necessarily mean there is a problem, they are often early warning indicators of emerging issues. Thus, taking the recommended action is advised.
 
+**Resource Usage Alerts**
```
Comment (ran-huang) [inline tidb-cloud/monitor-built-in-alerting.md:81]:
```suggestion
### Resource usage alerts

```

File: tidb-cloud/monitor-built-in-alerting.md [hunk 2]
```diff
@@ -88,12 +89,27 @@ The following table provides the TiDB Cloud built-in alert conditions and the co
...
+| Max memory utilization across TiKV nodes exceeded 70% for 10 minutes | Consider checking if there is any [Hotspot](https://docs.pingcap.com/tidbcloud/tidb-cloud-sql-tuning-overview#hotspot-issues) in the cluster or increasing the node number or node size for TiKV to reduce the memory usage percentage of the current workload. |
+| Max CPU utilization across TiDB nodes exceeded 80% for 10 minutes | Consider checking if there is any [Hotspot](https://docs.pingcap.com/tidbcloud/tidb-cloud-sql-tuning-overview#hotspot-issues) in the cluster or increasing the node number or node size for TiDB to reduce the CPU usage percentage of the current workload. |
+| Max CPU utilization across TiKV nodes exceeded 80% for 10 minutes | Consider checking if there is any [Hotspot](https://docs.pingcap.com/tidbcloud/tidb-cloud-sql-tuning-overview#hotspot-issues) in the cluster or increasing the node number or node size for TiKV to reduce the CPU usage percentage of the current workload. |
+
+**Data Migration Alerts**
```
Comment (ran-huang) [inline tidb-cloud/monitor-built-in-alerting.md:97]:
```suggestion
### Data migration alerts

```

File: tidb-cloud/monitor-built-in-alerting.md [hunk 3]
```diff
@@ -88,12 +89,27 @@ The following table provides the TiDB Cloud built-in alert conditions and the co
...
 | Data migration job met error during data import | Check the error and see [Troubleshoot data migration](/tidb-cloud/tidb-cloud-dm-precheck-and-troubleshooting.md#migration-errors-and-solutions) for help. |
 | Data migration job met error during incremental migration | Check the error and see [Troubleshoot data migration](/tidb-cloud/tidb-cloud-dm-precheck-and-troubleshooting.md#migration-errors-and-solutions) for help. |
 | Data migration job has been paused for more than 6 hours during incremental migration | Data migration job has been paused for more than 6 hours during data incremental migration. The binlog in the upstream database might be purged (depending on your database binlog purge strategy) and might cause incremental migration to fail. See [Troubleshoot data migration](/tidb-cloud/tidb-cloud-dm-precheck-and-troubleshooting.md#migration-errors-and-solutions) for help. |
 | Replication lag is larger than 10 minutes and still increasing for more than 20 minutes | See [Troubleshoot data migration](/tidb-cloud/tidb-cloud-dm-precheck-and-troubleshooting.md#migration-errors-and-solutions) for help. |
 
+**Changefeed Alerts**
```
Comment (ran-huang) [inline tidb-cloud/monitor-built-in-alerting.md:106]:
```suggestion
### Changefeed alerts

```

File: tidb-cloud/monitor-built-in-alerting.md [hunk 4]
```diff
@@ -88,12 +89,27 @@ The following table provides the TiDB Cloud built-in alert conditions and the co
 | Total TiFlash node CPU utilization exceeded 80% for 10 minutes | Consider increasing the node number or node size for TiFlash to reduce the CPU usage percentage of the current workload. |
 | TiKV storage utilization exceeds 80% | Consider increasing the node number or node storage size for TiKV to increase your storage capacity. |
 | TiFlash storage utilization exceeds 80% | Consider increasing the node number or node storage size for TiFlash to increase your storage capacity. |
-| Data migration job met error during data export | Check the error and see [Troubleshoot data migration](/tidb-cloud/tidb-cloud-dm-precheck-and-troubleshooting.md#migration-errors-and-solutions) for help.  |
+| Max memory utilization across TiDB Nodes exceeded 70% for 10 minutes | Consider checking if there is any [Hotspot](https://docs.pingcap.com/tidbcloud/tidb-cloud-sql-tuning-overview#hotspot-issues) in the cluster or increasing the node number or node size for TiDB to reduce the memory usage percentage of the current workload. |
```
Comment (ran-huang) [inline tidb-cloud/monitor-built-in-alerting.md:92]:
```suggestion
| Max memory utilization across TiDB nodes exceeded 70% for 10 minutes | Consider checking if there is any [hotspot](/tidb-cloud/tidb-cloud-sql-tuning-overview.md#hotspot-issues) in the cluster or increasing the node number or node size for TiDB to reduce the memory usage percentage of the current workload. |
```

File: tidb-cloud/monitor-built-in-alerting.md [hunk 5]
```diff
@@ -88,12 +89,27 @@ The following table provides the TiDB Cloud built-in alert conditions and the co
...
+| Max memory utilization across TiKV nodes exceeded 70% for 10 minutes | Consider checking if there is any [Hotspot](https://docs.pingcap.com/tidbcloud/tidb-cloud-sql-tuning-overview#hotspot-issues) in the cluster or increasing the node number or node size for TiKV to reduce the memory usage percentage of the current workload. |
+| Max CPU utilization across TiDB nodes exceeded 80% for 10 minutes | Consider checking if there is any [Hotspot](https://docs.pingcap.com/tidbcloud/tidb-cloud-sql-tuning-overview#hotspot-issues) in the cluster or increasing the node number or node size for TiDB to reduce the CPU usage percentage of the current workload. |
+| Max CPU utilization across TiKV nodes exceeded 80% for 10 minutes | Consider checking if there is any [Hotspot](https://docs.pingcap.com/tidbcloud/tidb-cloud-sql-tuning-overview#hotspot-issues) in the cluster or increasing the node number or node size for TiKV to reduce the CPU usage percentage of the current workload. |
```
Comment (ran-huang) [inline tidb-cloud/monitor-built-in-alerting.md:95]:
```suggestion
| Max memory utilization across TiKV nodes exceeded 70% for 10 minutes | Consider checking if there is any [hotspot](/tidb-cloud/tidb-cloud-sql-tuning-overview.md#hotspot-issues) in the cluster or increasing the node number or node size for TiKV to reduce the memory usage percentage of the current workload. |
| Max CPU utilization across TiDB nodes exceeded 80% for 10 minutes | Consider checking if there is any [hotspot](/tidb-cloud/tidb-cloud-sql-tuning-overview.md#hotspot-issues) in the cluster or increasing the node number or node size for TiDB to reduce the CPU usage percentage of the current workload. |
| Max CPU utilization across TiKV nodes exceeded 80% for 10 minutes | Consider checking if there is any [hotspot](/tidb-cloud/tidb-cloud-sql-tuning-overview.md#hotspot-issues) in the cluster or increasing the node number or node size for TiKV to reduce the CPU usage percentage of the current workload. |
```

File: tidb-cloud/monitor-built-in-alerting.md [hunk 6]
```diff
@@ -88,12 +89,27 @@ The following table provides the TiDB Cloud built-in alert conditions and the co
...
 | Data migration job met error during incremental migration | Check the error and see [Troubleshoot data migration](/tidb-cloud/tidb-cloud-dm-precheck-and-troubleshooting.md#migration-errors-and-solutions) for help. |
 | Data migration job has been paused for more than 6 hours during incremental migration | Data migration job has been paused for more than 6 hours during data incremental migration. The binlog in the upstream database might be purged (depending on your database binlog purge strategy) and might cause incremental migration to fail. See [Troubleshoot data migration](/tidb-cloud/tidb-cloud-dm-precheck-and-troubleshooting.md#migration-errors-and-solutions) for help. |
 | Replication lag is larger than 10 minutes and still increasing for more than 20 minutes | See [Troubleshoot data migration](/tidb-cloud/tidb-cloud-dm-precheck-and-troubleshooting.md#migration-errors-and-solutions) for help. |
 
+**Changefeed Alerts**
+| Condition | Recommended Action |
+|:--- |:--- |
+| Changefeed processor checkpoint delay more than 600 seconds | Check if the downstream system and network configuration are normal, and also exclude the possibility of an indexed table.  |
```
Comment (ran-huang) [inline tidb-cloud/monitor-built-in-alerting.md:109]:
```suggestion
| Changefeed processor checkpoint delay more than 600 seconds | Check if the downstream system and network configuration are functioning normally, and rule out the possibility of an indexed table.  |
```

File: tidb-cloud/monitor-built-in-alerting.md [hunk 7]
```diff
@@ -88,12 +89,27 @@ The following table provides the TiDB Cloud built-in alert conditions and the co
...
+
 > **Note:**
 >
 > - "cluster ABC" and "project XYZ" in the **Recommended Action** column are example names for reference.
+> - The thresholds of the alerts can be edited on TiDB Cloud Console. 
+> - Some alert rule is defualt disabled. User can enable them as needed.
```
Comment (ran-huang) [inline tidb-cloud/monitor-built-in-alerting.md:115]:
```suggestion
> - You can edit the thresholds of the alerts on the TiDB Cloud console. 
> - Some alert rules are disabled by default. You can enable them as needed. 
```

File: tidb-cloud/monitor-built-in-alerting.md [hunk 8]
```diff
@@ -88,12 +90,29 @@ The following table provides the TiDB Cloud built-in alert conditions and the co
...
 
+### Changefeed alerts
+
+| Condition | Recommended Action |
+|:--- |:--- |
+| Changefeed processor checkpoint delay more than 600 seconds | Check if the downstream system and network configuration are functioning normally, and rule out the possibility of an indexed table.  |
+
 > **Note:**
 >
 > - "cluster ABC" and "project XYZ" in the **Recommended Action** column are example names for reference.
```
Comment (hfxsd) [inline tidb-cloud/monitor-built-in-alerting.md:116]: There are no "cluster ABC" and "project XYZ" in this doc.
```suggestion
```

File: tidb-cloud/monitor-built-in-alerting.md [hunk 9]
```diff
@@ -88,12 +90,29 @@ The following table provides the TiDB Cloud built-in alert conditions and the co
...
 | Data migration job has been paused for more than 6 hours during incremental migration | Data migration job has been paused for more than 6 hours during data incremental migration. The binlog in the upstream database might be purged (depending on your database binlog purge strategy) and might cause incremental migration to fail. See [Troubleshoot data migration](/tidb-cloud/tidb-cloud-dm-precheck-and-troubleshooting.md#migration-errors-and-solutions) for help. |
 | Replication lag is larger than 10 minutes and still increasing for more than 20 minutes | See [Troubleshoot data migration](/tidb-cloud/tidb-cloud-dm-precheck-and-troubleshooting.md#migration-errors-and-solutions) for help. |
 
+### Changefeed alerts
+
+| Condition | Recommended Action |
+|:--- |:--- |
+| Changefeed processor checkpoint delay more than 600 seconds | Check if the downstream system and network configuration are functioning normally, and rule out the possibility of an indexed table.  |
+
 > **Note:**
```
Comment (hfxsd) [inline tidb-cloud/monitor-built-in-alerting.md:114]: Move this note to L77? Seems it applies to all alerts, not only changefeed alerts.

File: tidb-cloud/monitor-built-in-alerting.md [hunk 10]
```diff
@@ -76,7 +76,11 @@ The following table provides the TiDB Cloud built-in alert conditions and the co
 
 > **Note:**
 >
-> Although these alert conditions do not necessarily mean there is a problem, they are often early warning indicators of emerging issues. Thus, taking the recommended action is advised.
+> - Although these alert conditions do not necessarily mean there is a problem, they are often early warning indicators of emerging issues. Thus, taking the recommended action is advised.
```
Comment (ran-huang) [inline tidb-cloud/monitor-built-in-alerting.md:79]:
```suggestion
> - While these alert conditions do not necessarily mean there is a problem, they are often early warning indicators of emerging issues. Therefore, taking the recommended action is advised.
```

**Review by hfxsd — APPROVED** [review]

File: tidb-cloud/monitor-built-in-alerting.md [hunk 11]
```diff
@@ -88,12 +92,25 @@ The following table provides the TiDB Cloud built-in alert conditions and the co
...
-> **Note:**
->
-> - "cluster ABC" and "project XYZ" in the **Recommended Action** column are example names for reference.
+### Changefeed alerts
+
+| Condition | Recommended Action |
+|:--- |:--- |
+| Changefeed processor checkpoint delay more than 600 seconds | Check if the downstream system and network configuration are functioning normally, and rule out the possibility of an indexed table.  |
+
+
```
Comment (ran-huang) [inline tidb-cloud/monitor-built-in-alerting.md:116]:
```suggestion
```

**Pull request merged**